src/main/kotlin/app/revanced/patches/youtube/misc/fix/backtoexit/fingerprints/RecyclerViewTopScrollingFingerprint.kt

```diff
@@ -0,0 +1,16 @@
+package app.revanced.patches.youtube.misc.fix.backtoexit.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.Opcode
+
+object RecyclerViewTopScrollingFingerprint : MethodFingerprint(
+    opcodes = listOf(
+        Opcode.INVOKE_INTERFACE,
+        Opcode.MOVE_RESULT_OBJECT,
+        Opcode.CHECK_CAST,
+        Opcode.CONST_4,
+        Opcode.INVOKE_VIRTUAL,
+        Opcode.GOTO,
+        Opcode.IGET_OBJECT,
```

Comment:
This pattern is likely not gonna be unique. Strengthen the fingerprint without making it break easily.